.oidc_env.sample.json

```diff
@@ -0,0 +1,11 @@
+{
+  "google": {
+    "discovery_document_uri": "https://accounts.google.com/.well-known/openid-configuration",
+    "client_id": "CLIENT_ID",
+    "client_secret": "CLIENT_SECRET",
+    "redirect_uri": "https://example.com/session",
```

Comment:
Would it be more clear to use something like `https://firezone.example.com/auth/oidc/google/callback` so the user is aware of the path format requirement?